mysql-test/suite/galera/t/galera_gcs_fragment.test

```diff
@@ -1,14 +1,12 @@
+#
 # Test fragmentation over configuration changes
---source include/galera_cluster.inc
---source include/have_innodb.inc
-<<<<<<< HEAD
+#
+
 # This tc set debug sync point and so need galera library build in debug mode
 --source include/have_debug.inc
```

Comment:
Isn't it enough to have incoming change only? (galera_have_debug_sync.inc)

Reply:
This cosmetic change was made to differentiate between the test header and the comments.

The test shall look like below, if is left as is.
```
    1 # Test fragmentation over configuration changes
    2 # This tc set debug sync point and so need galera library build in debug mode
    3 --source include/have_debug.inc
    4 --source suite/galera/include/galera_have_debug_sync.inc
```